react/features/toolbox/components/native/ScreenSharingButton.js

```diff
@@ -15,4 +19,22 @@ const ScreenSharingButton = props => (
     </>
 );
 
-export default ScreenSharingButton;
+/**
+ * Maps (parts of) the redux state to the associated props for the
+ * {@code ScreenSharingButton} component.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     enabled: boolean,
+ * }}
+ */
+function _mapStateToProps(state): Object {
+    const disabled = state['features/base/audio-only'].enabled;
```

Comment:
This seems a bit weird to me, it sounds like it should either be negated, or it should be renamed to enabled.